### DISCUSSION TITLE
Pimcore Bundles 6.9.6??

### DISCUSSION THREAD

**Discussion by justbane** [pimcore/pimcore#10059] [Q&A / Support]
I see there is a new package for x10 - what about for those of us stuck on 6.9.6? How do we generate bundles? It seems the bundle generate command has been removed.

**Comment by NiklasBr** [comment]
[The docs for 6.9 are still available](https://pimcore.com/docs/pimcore/6.9/index.html), here is a [direct link to bundle section](https://pimcore.com/docs/pimcore/6.9/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/index.html), but the select on the top right sometimes does not include it.
